### PR TITLE
WIP Norne numerical aquifer

### PR DESCRIPTION
**Pull request by pgdr**
WIP

### CODE REVIEW

File: python/tests/core/ecl/test_grav.py [hunk 1]
```diff
@@ -22,4 +22,14 @@ def test_create(self):
 
             grav = EclGrav( self.grid , self.init )
 
-
+    def test_aquifer(self):
+        path = '/private/pgdr/opm/opm-data/norne/opm-simulation-reference/NORNE_ATW2013.'
+        grid = EclGrid(path+'EGRID')
+        print(repr(grid))
+        init = EclFile(path+'FINIT')
+        self.assertTrue(init.has_kw('AQUIFERN'))
```
Comment (joakim-hove) [inline python/tests/core/ecl/test_grav.py]: More Pythonic: `self.assertTrue( "AQUIFERN" in init )`

File: python/tests/core/ecl/test_grav.py [hunk 2]
```diff
@@ -22,4 +22,14 @@ def test_create(self):
 
             grav = EclGrav( self.grid , self.init )
 
-
+    def test_aquifer(self):
+        path = '/private/pgdr/opm/opm-data/norne/opm-simulation-reference/NORNE_ATW2013.'
+        grid = EclGrid(path+'EGRID')
+        print(repr(grid))
+        init = EclFile(path+'FINIT')
```
Comment (joakim-hove) [inline python/tests/core/ecl/test_grav.py]: `EclFile(path + "INIT")`

**Comment by joakim-hove** [comment]
Should this be merged?

**Comment by pgdr** [comment]
No, I don't think it's needed anymore.

**Pull request closed without merging**